dev-workflow-ui/cms/cms_de.yaml

```diff
@@ -227,6 +227,8 @@ statistics:
   allTasks: Alle Aufgaben
   casesByState: Fälle nach Status
   casesStatistics: Fall Statistik
+  chartResolution: Diagramm Auflösung
+  every: —
```

Comment:
I don't think that we need to translate this right? If it is a '-' then it should be like this for every language